streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/master/resourcemanager/strategy/ResourceAssignStrategy.java

```diff
@@ -0,0 +1,29 @@
+package org.ray.streaming.runtime.master.resourcemanager.strategy;
+
+import java.util.List;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionGraph;
+import org.ray.streaming.runtime.core.resource.Container;
+import org.ray.streaming.runtime.master.resourcemanager.ResourceAssignmentView;
+
+/**
+ * The ResourceAssignStrategy responsible assign {@link Container} to
+ * {@link org.ray.streaming.runtime.core.graph.executiongraph.ExecutionVertex}.
```

Comment:
Move the full qualified class name into import.
e.g.
import xxx.xxx.A
{@link A}

Reply:
Full name more better?